src/isar/state_machine/states/initialize.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import time
+from typing import TYPE_CHECKING, Callable, Optional
+
+from injector import inject
+from transitions import State
+
+from isar.services.utilities.threaded_request import (
+    ThreadedRequest,
+    ThreadedRequestNotFinishedError,
+)
+from robot_interface.models.exceptions import RobotException
+
+if TYPE_CHECKING:
+    from isar.state_machine.state_machine import StateMachine
+
+
+class Initialize(State):
+    @inject
+    def __init__(self, state_machine: "StateMachine"):
+        super().__init__(name="initialize", on_enter=self.start, on_exit=self.stop)
+        self.state_machine: "StateMachine" = state_machine
+
+        self.logger = logging.getLogger("state_machine")
+        self.initialize_thread: Optional[ThreadedRequest] = None
```

Comment:
If the state is already called `Initialize`, shouldn't the thread simply be named `thread` ?
`initialize_thread` sounds like a function name to me.

Reply:
The name reflects the name of the function (see the other states for example), not the state.